osgeo_importer/importers.py

```diff
@@ -223,14 +225,16 @@ def import_file(self, *args, **kwargs):
 
         # Add index for any layers configured by name
         for layer_configuration in configuration_options:
-            lookup = 'layername'
-
-            if lookup not in layer_configuration and 'index' in layer_configuration:
+            if 'layer_name' in layer_configuration:
```

Comment:
`lookup` needs to be declared with a default value or the for loop below will throw an exception if both `if` statements are passed.